src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private static final int DIVIDER_FOR_AREA = 2;
+    private int upperSide;
+    private int lowerSide;
+    private int height;
+
+    public IsoscelesTrapezoid(String color, int upbothside, int downbothside, int height) {
+        super(color);
+        this.upperSide = upbothside;
+        this.lowerSide = downbothside;
+        this.height = height;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: IsoscelesTrapezoid");
+        System.out.println("Color: " + getColor());
+        System.out.println("Up both side length: " + upperSide);
+        System.out.println("Down both side length: " + lowerSide);
+        System.out.println("Height length: " + height);
```

Comment:
Consider using a StringBuilder for concatenating multiple strings, especially in a method that could be called multiple times, like 'draw()'. This can improve performance by reducing the number of temporary string objects created.